File: frontend/src/model/Ad.ts
export type Ad = {
  id: number;
  title: string;
  description: string;
  address: string;
  price: number;
  roomCount: number;
  area: number;
  createdAt: Date;
  owner: {
    fullName: string;
    userName: string;
    email: string;
    phoneNumber: string;
  };
  image: string; // Base64
};

File: RealEstate/Domain/Models/Ad.cs
using System.Drawing;

namespace Domain.Models
{
    public class Ad
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
        public int Price { get; set; }
        public int RoomCount { get; set; }
        public int Area { get; set; }
        public DateTime CreatedAt { get; set; }
        public string Image { get; set; }    // Use Base64 representation
        public User Owner { get; set; }
        public bool Highlighted { get; set; }
    }
}